frontend/src/content/news.json

```
@@ -1,9 +1,14 @@
 {
   "2022": [
+    {
+      "date": "2022-06-01",
+      "title": "Metabolic Atlas 2.6 adds a feature to visualise models updates",
```

Ah sorry @inghylt the previous suggestion introduced a typo:
```suggestion
      "title": "Metabolic Atlas 2.6 adds a feature to visualise model updates",
```

---

Yes that's great, thank you :pray: